libsolidity/ast/ASTAnnotations.h

```diff
@@ -149,7 +150,7 @@ struct StructDeclarationAnnotation: TypeDeclarationAnnotation
 struct ContractDefinitionAnnotation: TypeDeclarationAnnotation, StructurallyDocumentedAnnotation
 {
 	/// List of functions and modifiers without a body. Can also contain functions from base classes.
-	std::vector<Declaration const*> unimplementedDeclarations;
+	std::optional<std::vector<Declaration const*>> unimplementedDeclarations;
```

Comment:
Why optional and not setonce?

Reply:
Because it's filled up in 2 different functions.. though if I can actually initialize it once and add content then `SetOnce` would work

Reply:
but `SetOnce` only allows `const` access so it has to be `optional`